application/templates/static_site/static_pages/cookies.html

```diff
@@ -58,28 +58,28 @@ <h2 class="govuk-heading-s">Measuring website usage (Google Analytics)</h2>
 
         <p class="govuk-body">Google Analytics sets the following cookies:</p>
 
-        <table class="cookie-info govuk-!-margin-bottom-8">
-          <thead>
-            <tr>
-              <th class="name">Name</th>
-              <th>Purpose</th>
-              <th class="expires">Expires</th>
+        <table class="govuk-table govuk-!-margin-bottom-8">
+          <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+              <th class="govuk-table__header" scope="col">Name</th>
+              <th class="govuk-table__header" scope="col">Purpose</th>
+              <th class="govuk-table__header" scope="col">Expires</th>
             </tr>
           </thead>
-          <tbody>
-            <tr>
-              <td>_ga</td>
-              <td>This helps us count how many people visit this service by tracking if you’ve visited before</td>
-              <td>2 years</td>
+          <tbody class="govuk-table__body">
+            <tr class="govuk-table__row">
+              <td class="govuk-table__header" scope="row">_ga</td>
+              <td class="govuk-table__cell">This helps us count how many people visit this service by tracking if you’ve visited before</td>
+              <td class="govuk-table__cell">2 years</td>
```

Comment:
Could you add a `&nbsp;` (non-breaking space) between the numeral and the word "years" here please? Same with the ones below.